ocs_ci/helpers/dr_helpers_ui.py

```
@@ -1838,3 +1838,879 @@ def navigate_using_fleet_virtualization(acm_obj):
+
+    # Clear any existing filters (button only appears when filters are active)
+    try:
+        clear_filter_present = acm_obj.check_element_presence(
+            (acm_loc["clear-filter"][1], acm_loc["clear-filter"][0]),
+            timeout=5,
+        )
+        if clear_filter_present:
+            log.info("Clearing existing filters")
+            acm_obj.do_click(acm_loc["clear-filter"])
+    except Exception:
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)